doc/chapters/api/index.rst

```diff
@@ -1,21 +1,10 @@
-
-.. _api:
-
-API
-***
-
-This chapter documents the DBus and C API for interacting with the SoftwareContainerAgent. This is
-mostly useful for integrators.
-
-.. TODO::
-    This chapter is still to be written properly. Some bits and pieces have
-    been but they are under development.
-
```

Comment:
I agree with tobsan, this chapter should be used to document all public API's imho. For now, that only means the D-Bus API, but I'd like to have a top level chapter. Also, this chapter is pointed to by other chapters using the top level reference so if you insist on removing it you need to re-point the api references in the other chapters.